### DISCUSSION TITLE
User Manager

### DISCUSSION THREAD

**Discussion by julisubu** [DouglasNeuroInformatics/OpenDataCapture#969] [Q&A] [answered]
How can I create and manage accounts to assign roles and permissions?

**Comment by joshunrau** [comment]
Hi! I am working on implementing a feature for this right now. I will let you know as soon as it is done!

**Reply by joshunrau** [reply]
This will be soon (within a few days max)

**Reply by joshunrau** [reply, marked as answer]
This is now implemented in v1.3.0. Please update your instance and you should be able to access via the admin account. Any feedback appreciated.

**Reply by joshunrau** [reply]
Also, please note that there will be changes in the future to allow for updating groups and users. For now, you can only create and delete users and groups.

**Comment by julisubu** [comment]
How could we modify or delete instruments?, it would be a useful function within the administrator profile

**Reply by gdevenyi** [reply]
Instruments cannot be deleted, as there is potentially data in the DB which was generated from a given instrument. Modification is done by uploading a new internal.edition of an instrument, which keeps the data associated with the instrument, while allowing for upgraded versions.
